proto/compliance/tx.proto

```diff
@@ -85,4 +86,33 @@ message MsgProvisionModel {
 message MsgProvisionModelResponse {
 }
 
+message MsgUpdateComplianceInfo {
+  string creator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
+  int32 vid = 2 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""]; 
+  int32 pid = 3 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""];
+  uint32 softwareVersion = 4;
+  string softwareVersionString = 5 [(gogoproto.moretags) = "validate:\"omitempty,max=64\""];
+  string certificationType = 6 [(gogoproto.moretags) = "validate:\"required,max=64\""];
+  uint32 cDVersionNumber = 7 [(gogoproto.moretags) = "validate:\"omitempty,gte=0,lte=65535\""]; 
```

Comment:
This field may be immutable as well. Currently there are no checks regarding cDVersionNumber when Compliance is created, though we need to check if it must be equal to cDVersionNumber from the ModelVersion. If it must be equal, then cDVersionNumber must be immutable and removed from here.